client/my-sites/plans/plan-overview/plan-features/index.jsx

```diff
@@ -81,7 +81,7 @@ const PlanFeatures = React.createClass( {
 					willBeRemoved={ willBeRemoved } />
 
 				<PlanFeature
-					description={ this.translate( 'You can live chat with our happiness engineers anytime you need.' ) }
+					description={ this.translate( 'You can contact our happiness engineers anytime you need.' ) }
```

Comment:
Based on the issue, it sounds like we should only be updating the copy for users that on the premium trial. We should be able to include two separate descriptions based on `isBusiness` / `isPremium` here pretty easily.